be/src/olap/accept_null_predicate.h

```diff
@@ -52,8 +52,37 @@ class AcceptNullPredicate : public ColumnPredicate {
 
     uint16_t evaluate(const vectorized::IColumn& column, uint16_t* sel,
                       uint16_t size) const override {
-        LOG(FATAL) << "evaluate without flags not supported";
-        // return _nested->evaluate(column, sel, size);
+        // call nested predicate evaluate
+        uint16_t new_size = _nested->evaluate(column, sel, size);
+
+        // process NULL values
+        if (column.has_null() && new_size < size) {
+            // use a sorted set to sotre selected row idxs
+            std::set<uint16_t> selected_rows;
```

Comment:
it may cost a lot of time to insert the null rows and selected row into the set, because it need to sort.